Formula/ghc@8.6.rb

```diff
@@ -33,6 +34,9 @@ class GhcAT86 < Formula
     sha256 "dfc1bdb1d303a87a8552aa17f5b080e61351f2823c2b99071ec23d0837422169"
   end
 
+  # Fix for Catalina compatibility https://gitlab.haskell.org/ghc/ghc/issues/17353
+  patch :DATA
```

Comment:
CC @fxcoudert to check this is as expected as you're in that thread 😁

Reply:
Patch looks OK to me